sdks/python/build.gradle

```diff
@@ -197,13 +197,13 @@ task directRunnerIT(dependsOn: 'installGcpTest') {
 //    ./gradlew :beam-sdks-python:portableWordCount -PjobEndpoint=localhost:8099
 //
 task portableWordCount {
-  dependsOn portableWordCountTask('portableWordCountExample', project.hasProperty("streaming"))
+  dependsOn portableWordCountTask('portableWordCountExample', project.hasProperty("streaming"), project.hasProperty("crossLanguage"))
```

Comment:
@ihji thanks for the contribution. The PR title was a bit misleading :) 

I wonder if we should clearer delineate the simple, portable from the cross language example WRT their dependencies. There is no need to build a java container or have the expansion service for the former and users will likely care about that. @mxm WDYT?

Reply:
I agree, the Java SDK Harness dependency shouldn't be required for the regular portable wordcount. That can be fixed. It looks like @ihji wanted to avoid code duplication. Could we extract all configuration and make them task parameters? That way we would get rid of the `if cross_language:` statements.